MBProgressHUD.podspec

```diff
@@ -12,8 +12,8 @@ Pod::Spec.new do |s|
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.author       = { 'Matej Bukovinski' => 'matej@bukovinski.com' }
   s.source       = { :git => "https://github.com/matej/MBProgressHUD.git", :tag => s.version.to_s }
-  s.ios.deployment_target = '9.0'
-  s.tvos.deployment_target = '9.0'
+  s.ios.deployment_target = '12.0'
```

Comment:
Hey @jarrodlombardo-EventBase, can you add the `resource_bundles` to include the `PrivacyInfo.xcprivacy`?
This will help to include the file in the framework.

```
   s.resource_bundles = {'MBProgressHUD' => ['PrivacyInfo.xcprivacy']}
```

Reply:
We don't use cocoapods at Eventbase, so I don't trust myself to make that change. You can open a PR into https://github.com/eventbasedev/MBProgressHUD/tree/master and my team can look at it and then it would become part of this PR.

Reply:
Sure, created the [PR](https://github.com/eventbasedev/MBProgressHUD/pull/3) in the eventbasedev. Thank you.

Reply:
that PR is merged. :)